plugins/modules/vmware_guest_ft.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+DOCUMENTATION = r'''
+module: vmware_guest_ft
+short_description: Fault Tolerance for specified VM.
+description: This module is used for enable or disable the Fault Tolerance for an existing  Virtual Machine.
+author:
+     - Christian Neugum (@digifuchsi)
+     - Valentin Yonev (@valentinJonev)
+notes:
+    - Tested on vSphere 5.5, 6.5, 7.0
+requirements:
+    - "python >= 2.6"
+    - PyVmomi
+options:
+    virtual_machine_name:
+        description: Name of the Virtual Machine
+        type: str
+        required: True
+    state:
+        description:
+        - present - represents that Fault Tolerance should be enabled
+        - absent - disables the Fault Tolerance
+        choices: [ present, absent ]
+        default: present
+        type: str
+'''
+
+EXAMPLES = r'''
+- name: enable FT on VM
+  vmware_guest_ft:
+    virtual_machine_name: VM1
+    state: present
+  delegate_to: localhost
+'''
+
+import time
+
+__metaclass__ = type
+
+
+try:
+    from pyVim.task import WaitForTask
+    from pyVmomi import vim, vmodl, PyVmomi
+    # from tools import tasks
+    from ansible.module_utils._text import to_native
+except ImportError as e:
+    raise e
```

Comment:
```suggestion
try:
    from pyVim.task import WaitForTask
    from pyVmomi import vim, vmodl, PyVmomi
    # from tools import tasks
    from ansible.module_utils._text import to_native
except ImportError:
    pass
```

Please ignore the import error here. That's what the other other modules do, since `PyVmomi` already handles this. I think this will help to make the sanity tests happy.

Apropos making the sanity tests happy, you also have to add the GPLv3 license header.